packages/example-completeness/src/PriceNormalizer.js

```diff
@@ -18,8 +16,12 @@ exports.normalize = function(object) {
       } else {
         return `@.${identifier}=="${jp.query(object, `$.${identifier}`)}"`;
       }
+
+      return null;
```

Comment:
Otherwise this returns undefined which is considered a non-consistent return which can yield in very odd bugs.